pkg/storageincentives/agent.go

```diff
@@ -413,13 +441,18 @@ func (a *Agent) Close() error {
 	}
 }
 
-func (s *Agent) wrapCommit(storageRadius uint8, sample []byte, key []byte) ([]byte, error) {
+func (a *Agent) wrapCommit(storageRadius uint8, sample []byte, key []byte) ([]byte, error) {
 
 	storageRadiusByte := []byte{storageRadius}
 
-	data := append(s.overlay.Bytes(), storageRadiusByte...)
+	data := append(a.overlay.Bytes(), storageRadiusByte...)
 	data = append(data, sample...)
 	data = append(data, key...)
 
 	return crypto.LegacyKeccak256(data)
 }
+
+// Status returns the node status
+func (a *Agent) Status() (status NodeStatus, err error) {
```

Comment:
The named return values are not needed here.

Reply:
Correct.. Forgot to remove it